flux-cli/src/main/java/com/marklogic/flux/impl/importdata/ImportFilesCommand.java

```diff
@@ -24,21 +24,38 @@ public class ImportFilesCommand extends AbstractImportFilesCommand<GenericFilesI
     @CommandLine.Mixin
     private WriteGenericDocumentsParams writeParams = new WriteGenericDocumentsParams();
 
+    @CommandLine.Option(
+        names = "--streaming",
+        description = "Causes files to be read from their source directly to MarkLogic. Intended for importing large " +
```

Comment:
I'm curious if there's a significant drawback to using streaming even if it isn't strictly necessary.

Reply:
The main point in the upcoming docs will be that it's necessarily limited to a document per request. That's true on both write and read.